src/constants.ts

```diff
@@ -39,3 +43,9 @@ export const PRIVACY_POLICY_LINKS = {
   sv:
     'https://www.hel.fi/static/liitteet-2019/Kaupunginkanslia/Rekisteriselosteet/Kuva/Personregister%20Tj%C3%A4nstepalett.pdf',
 };
+
+export const EMPTY_LOCALISED_OBJECT: LocalisedObject = {
+  [SUPPORT_LANGUAGES.FI]: '',
+  [SUPPORT_LANGUAGES.SV]: '',
+  [SUPPORT_LANGUAGES.EN]: '',
+};
```

Comment:
Can be problematic if this is used in many places if data structures are mutated

Reply:
fixed with Freeze

Reply:
Actually decided to create copy instead, freezing shows that formik does not seem to mutate it because no error thrown, but copying is correct solution I think.